ceph.spec.in

```diff
@@ -457,6 +447,34 @@ module derived from Calamari) and expose CLI hooks.  ceph-mgr gathers
 the cluster maps, the daemon metadata, and performance counters, and
 exposes all these to the python modules.
 
+%package mgr-dashboard
+Summary:        Ceph Manager Dashboard
+BuildArch:      noarch
+%if 0%{?suse_version}
+Group:          System/Filesystems
+%endif
+Requires:       ceph-mgr = %{_epoch_prefix}%{version}-%{release}
+%if 0%{?fedora} || 0%{?rhel}
+Requires:       python%{_python_buildid}-cherrypy
+Requires:       python%{_python_buildid}-jwt
+Requires:       python%{_python_buildid}-routes
+Requires:       python%{_python_buildid}-werkzeug
+Requires:       pyOpenSSL%{_python_buildid}
+Requires:       python%{_python_buildid}-bcrypt
+%endif
+%if 0%{?suse_version}
+Requires:       python%{_python_buildid}-CherryPy
+Requires:       python%{_python_buildid}-PyJWT
+Requires:       python%{_python_buildid}-Routes
+Requires:       python%{_python_buildid}-Werkzeug
+Requires:       python%{_python_buildid}-pyOpenSSL
+Requires:       python%{_python_buildid}-bcrypt
+%endif
+%description mgr-dashboard
```

Comment:
The Dashboard as shipped with Nautilus is much more than just visualizing information.

How about changing the description as follows:

> The Ceph Manager Dashboard is a manager plugin, providing a web-based application to monitor and manage many aspects of a Ceph Cluster and related components. See the Dashboard documentation at http://docs.ceph.com/ for details and a detailed feature overview.

Reply:
@LenzGr i revised the suggested description a little bit.

1. adapt to the convention of the debian/control and spec -- to start the descriptions with "This package provides a ceph-mgr plugin" and "ceph-mgr-dashboard is a manager plugin" respectively.
1. lower case "Cluster", as it's not a specific cluster, and to be consistent with other descriptions
1. add a new line in debian's description.